Makefile

```diff
@@ -406,8 +406,14 @@ e2e-test/teardown:
 	./test/e2e/setup/e2e_teardown.sh
 .PHONY: e2e-test/teardown
 
-e2e-test: e2e-test/teardown e2e-test/setup
+e2e-test:
+	@if [ -z "$$consumer_name" ] || [ -z "$$api_server" ] || [ -z "$$consumer_kubeconfig" ]; then \
```

Comment:
why need this judgement?

Reply:
By default, the e2e test will run on the KinD cluster by running `e2e-test/setup`. Here we can provide a way to run the  `e2e-test` on other platforms like `AKS` or `Openshift`. All we need to do is given the environment variables `consumer_name`, `api_server`, and `consumer_kubeconfig`.

I found it convenient during testing, but could this be considered over-engineering? Do you have any suggestions?

Reply:
maybe export a single environment to indicate whether need to setup the environment or use the existing environment. What do you think?

Reply:
Maybe we can add this feature in another PR, I removed it from the current.